includes/Modules/Analytics.php

```diff
@@ -1335,27 +1358,30 @@ function ( Google_Service_Analytics_Account $account ) {
 	 * Creates a new Analytics site request for the current site and given arguments.
 	 *
 	 * @since 1.0.0
+	 * @since n.e.x.t Added $dimension_filters
 	 *
 	 * @param array $args {
 	 *     Optional. Additional arguments.
 	 *
-	 *     @type array  $dimensions List of request dimensions. Default empty array.
-	 *     @type string $start_date Start date in 'Y-m-d' format. Default empty string.
-	 *     @type string $end_date   End date in 'Y-m-d' format. Default empty string.
-	 *     @type string $page       Specific page URL to filter by. Default empty string.
-	 *     @type int    $row_limit  Limit of rows to return. Default 100.
+	 *     @type array  $dimensions        List of request dimensions. Default empty array.
+	 *     @type array  $dimension_filters List of dimension filters for the specified request dimensions. Default empty array.
```

Comment:
Let's add the type here as it isn't quite clear that this argument requires `Google_Service_AnalyticsReporting_DimensionFilter` instances.

```suggestion
	 *     @type Google_Service_AnalyticsReporting_DimensionFilter[]  $dimension_filters List of dimension filter instances for the specified request dimensions. Default empty array.
```